05_get_stats_02_identifying_data.Rmd

```diff
@@ -19,56 +19,40 @@ Thus, although each of us generate data daily through the apps we use or through
 
 
-![Trump Tweets Blog Post](https://docs.google.com/presentation/d/1hin5y4jDZikoGLbIEnwnPhJybgD_FezWsh16QIu5C5o/export/png?id=1hin5y4jDZikoGLbIEnwnPhJybgD_FezWsh16QIu5C5o&pageid=g3ec882b221_0_155)
+![IMDB](https://docs.google.com/presentation/d/1hin5y4jDZikoGLbIEnwnPhJybgD_FezWsh16QIu5C5o/export/png?id=1hin5y4jDZikoGLbIEnwnPhJybgD_FezWsh16QIu5C5o&pageid=g3ec882b221_0_155)
 
 ### The Perfect Dataset: The Data We Want
 
 Once you determine your data science question of interest, it's helpful to **envision the perfect dataset**. This dataset likely doesn't exist in the real world, but our goal will be to get data that match this optimal dataset *as closely as possible*.
 
 Thus to answer the question
 
-> Do the tweets on the _@realDonaldTrump_ account differ between those posted using an iPhone vs. those posted using an Android? Are the Android tweets angrier and more negative?
+> What variables from IMDB predict the author's movie ratings?
 
 we would ideally want a dataset where each row is a different tweet with each row containing information on the following variables:
 
-* `date` - date the tweet was written
-* `time` - time the tweet went out
-* `os` - the operating system of where the tweet came from (Android or iPhone)
-* `author`- author of each tweet
-* `location` - where the author was when the tweet was written
-* `tweet` - what the tweet said
-* `censored` - whether or not the tweet was later deleted
-  * `censored_date` - date on which the tweet was deleted
-  * `censored_time` - time at which the tweet was deleted
-* `anger` - some measurement of how angry and negative the tweet is
-
-Additionally, in a perfect world, we'd have all the tweets ever tweeted from _@realDonaldTrump_.
+- TODO: Explaining the ideal variables in the dataset
 
 Having all these data in this format would enable us to answer the question of interest!
 
 ### The Data We Have
 
 *But*, the data we *have* are typically not *exactly* the data we *want*.
 
-In this case, David Robinson was able to use the `twitteR` package and the Twitter API to obtain a number of Donald Trump's tweets. Ultimately, the dataset he used for his analysis contained 628 tweets from an iPhone, and 762 tweets from an Android. That's not nearly *all* the tweets from this account; however, the Twitter API limits the number of tweets one can obtain. Already, the **data we have are not the optimal data we wanted**, but that's OK.
-
-Additionally, the data from the Twitter API does *not* include the `author`, the author's location, nor any information about deleted tweets. Again, the data we have are quite different from our optimal data.
-
-Finally, the dataset does not include any measure of how angry or negative the content of the tweets are. David will have to use the tweets to determine that during his analysis.
-
+- TODO: Explaining the variables we have
```

Comment:
This we can do easy enough if we decide to stick with this dataset as the example.

Reply:
If you could add this as it must have been deleted in the file I sent:

To make this analysis more robust and more generic, we would need to have the personal rating of a lot more movies (both number and variation) but also a lot more people. This would allow us to build a customized model for each person based on their preferences. 
Taking this analysis further,  we could gather more information about the viewer (age, sex, location, etc) and try to build a customized rating model for that profile of person.